diff --git a/node_modules/@ui-kitten/eva-icons/evaIcon.component.js b/node_modules/@ui-kitten/eva-icons/evaIcon.component.js
index a474974..1ce8060 100644
--- a/node_modules/@ui-kitten/eva-icons/evaIcon.component.js
+++ b/node_modules/@ui-kitten/eva-icons/evaIcon.component.js
@@ -1,24 +1,31 @@
-var __rest = (this && this.__rest) || function (s, e) {
+var __rest =
+  (this && this.__rest) ||
+  function(s, e) {
     var t = {};
-    for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p) && e.indexOf(p) < 0)
+    for (var p in s)
+      if (Object.prototype.hasOwnProperty.call(s, p) && e.indexOf(p) < 0)
         t[p] = s[p];
     if (s != null && typeof Object.getOwnPropertySymbols === "function")
-        for (var i = 0, p = Object.getOwnPropertySymbols(s); i < p.length; i++) {
-            if (e.indexOf(p[i]) < 0 && Object.prototype.propertyIsEnumerable.call(s, p[i]))
-                t[p[i]] = s[p[i]];
-        }
+      for (var i = 0, p = Object.getOwnPropertySymbols(s); i < p.length; i++) {
+        if (
+          e.indexOf(p[i]) < 0 &&
+          Object.prototype.propertyIsEnumerable.call(s, p[i])
+        )
+          t[p[i]] = s[p[i]];
+      }
     return t;
-};
-import React from 'react';
+  };
+import React from "react";
 export class EvaIcon {
-    constructor(content) {
-        this.content = content;
-    }
-    toReactElement(props) {
-        const Icon = this.content;
-        // @ts-ignore - Eva maps icon color to `tintColor`
-        const { tintColor } = props, restProps = __rest(props, ["tintColor"]);
-        return (<Icon fill={tintColor} {...restProps}/>);
-    }
+  constructor(content) {
+    this.content = content;
+  }
+  toReactElement(props) {
+    const Icon = this.content;
+    // @ts-ignore - Eva maps icon color to `tintColor`
+    const { tintColor } = props,
+      restProps = __rest(props, ["tintColor"]);
+    return React.createElement(Icon, { file: tintColor, ...restProps });
+  }
 }
 //# sourceMappingURL=evaIcon.component.js.map
